src/components/forms/TimePicker/TimePicker.stories.tsx

```diff
@@ -54,6 +54,7 @@ export const defaultTimePicker = (
     <TimePicker
       id="appointment-time"
       name="appointment-time"
+      label="Appointment Time"
```

Comment:
USWDS' example uses sentence case for this label, we should match their example:

```suggestion
      label="Appointment time"
```